build.cake

```diff
@@ -117,12 +118,14 @@ Task("__UpdateAssemblyVersionInformation")
     try {
         IEnumerable<string> outputLines;
         StartProcess(gitVersionPath, gitVersionSettings, out outputLines);
+        Information(outputLines);
 
         var output = string.Join("\n", outputLines);
         gitVersionOutput = Newtonsoft.Json.JsonConvert.DeserializeObject<Dictionary<string, object>>(output);
     }
-    catch
+    catch(Exception ex)
     {
+        Information(ex);
```

Comment:
I guess this is just for debugging why it's failing?

Reply:
Yes, Seems to be a bug in Gitversion tool